library/core/src/time.rs

```diff
@@ -697,6 +697,44 @@ impl Duration {
         }
     }
 
+    /// The checked version of [`from_secs_f64`].
+    ///
+    /// [`from_secs_f64`]: Duration::from_secs_f64
+    ///
+    /// This constructor will return `None` if `secs` is not finite, negative or overflows `Duration`.
```

Comment:
This documentation doesn't match the method's type signature. The method returns a `Result`, but this mentions `None`.

Reply:
This should be fixed now